developer/contributing/pull-requests.rst

```diff
@@ -32,67 +54,116 @@ If you are a member of the `Sulu organization`_ you should also:
 Template
 --------
 
-Template is divided in two parts: *tasks* (for developer) and *information*
-(for reviewer).
+When creating a pull request on GitHub, there is already a pre filled template
+for you. The template is divided in several parts.
 
-The information should include the following fields:
+Checklist
+`````````
+The pull request must include the following checklist at the top to ensure that
+contributions may be reviewed without needless feedback loops and that your
+contributions can be included into Sulu as quickly as possible:
 
-* **Fixed tickets**: **list of tickets**; Tickets to close after merge. These
-  should be of the form ``fixes #123, fixes #321``.
-* **Related PRs**: **link to PRs`**, PRs that are required for the current one
-* **BC breaks**: **none or description**; any compatibility breaking changes
-* **Documentation PR**: **link to PR**; Link to the documentation PR
-  (if applicable)
+.. code-block:: text
 
-.. note::
+    | Q                | A
+    | ---------------- | ---
+    | Bug fix?         | [yes|no]
+    | New feature?     | [yes|no]
+    | BC breaks?       | [yes|no]
+    | Deprecations?    | [yes|no]
+    | Tests pass?      | [yes|no]
```

Comment:
Do we actually need this line? Given that CI systems are integrated in GitHub, I think this is superfluous.